fluentrequest.go

```diff
@@ -28,8 +31,25 @@ func (r *fluentRequest) Method(method string) *fluentRequest {
 	return r
 }
 
+func (r *fluentRequest) Body(body io.Reader) *fluentRequest {
+	r.body = body
+
+	return r
+}
+
+func (r *fluentRequest) Header(header map[string][]string) *fluentRequest {
+	r.header = header
+
+	return r
+}
+
 func (r *fluentRequest) Run() (*http.Response, error) {
 
-	req, _ := http.NewRequest(r.method, r.url, nil)
+	req, _ := http.NewRequest(r.method, r.url, r.body)
+
+	if r.header != nil {
+		req.Header = r.header
+	}
```

Comment:
Is this nil verification really necessary? I did some tests and r.header had no problems in being nil